docs/user_guide/hopsworks/jupyter.rst

```diff
@@ -337,6 +337,22 @@ pull from a remote or push to a remote etc. For more complicated operations you
 
     Notebooks version control
 
+Debug Jupyter installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Jupyter is installed in the python environment of your project. This means that if a dependency of Jupyter is removed or an incorrect version is installed it may not work properly.
+If the environment ends up in a state where the installation may be compromised then an alert will be shown in the interface explaining the issue.
```

Comment:
>If the environment ends up in a state where the installation may be compromised

Maybe we can say something less worrisome like
`If the Python environment ends up in a state with conflicting libraries installed`